src/js/modules/media/reducer.js

```diff
@@ -23,7 +21,14 @@ const reducer = (state = { ...initialState }, action) => {
         ...state,
         isFetching: false,
         isFetched: true,
-        response: action.payload,
+        // replacing acc with state.media will block component updates
```

Comment:
actually, by deleting the `ADD_MEDIA` and all the other `ADD`s, it makes the homepage module make more sense